pkg/virtual/workspaces/registry/rest.go

```diff
@@ -215,17 +207,22 @@ func withoutGroupsWhenPersonal(user user.Info, scope string) user.Info {
 func (s *REST) extractOrg(user user.Info, ctx context.Context) (orgClusterName string, org *Org, err error) {
 	orgClusterName = ctx.Value(WorkspacesOrgKey).(string)
 
-	// Any user must has at least access to the content of the root org, in order to
-	// be able to fetch the list of the orgs he is member of
+	// Any user must have at least access to the content of the root org, in order to
```

Comment:
not really. Better: `A user must have at least access to the org **via** clusterworkspace/content **in** the root org, ....